nflow-explorer-ng/src/App.scss

```diff
@@ -2,5 +2,5 @@
   margin-left: auto;
   margin-right: auto;
   max-width: 1920px;
-  padding: var(--size-xl);
+  padding: var(--size-m);
```

Comment:
I think this would look visually better:
```
padding: 0;
```

Reply:
Or not to have padding line at all, same effect.

Reply:
It's in now. Though you're also welcome to make your own PRs for random pixel tinkerings ;)